src/Workspaces/Core/Portable/FindSymbols/Shared/AbstractSyntaxIndex_Persistence.cs

```diff
@@ -23,7 +23,7 @@ internal abstract partial class AbstractSyntaxIndex<TIndex>
     /// that we will not try to read previously cached data from a prior version of roslyn with a different format and
     /// will instead regenerate all the indices with the new format.
     /// </summary>
-    private static readonly Checksum s_serializationFormatChecksum = CodeAnalysis.Checksum.Create("45");
+    private static readonly Checksum s_serializationFormatChecksum = CodeAnalysis.Checksum.Create("46");
```

Comment:
>46 [](http://example.com/codeflow?start=99&length=2)

just making sure this is needed.

Reply:
yup.  changing what we emit into the index.